magik-checks/src/test/java/nl/ramsolutions/sw/checks/magik/fixers/UseValueCompareFixerTest.java

```diff
@@ -0,0 +1,146 @@
+package nl.ramsolutions.sw.checks.magik.fixers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import nl.ramsolutions.sw.magik.CodeAction;
+import nl.ramsolutions.sw.magik.MagikFile;
+import nl.ramsolutions.sw.magik.Position;
+import nl.ramsolutions.sw.magik.Range;
+import nl.ramsolutions.sw.magik.TextEdit;
+import org.junit.jupiter.api.Test;
+
+/** Tests for {@link UseValueCompareFixer}. */
+@SuppressWarnings("checkstyle:MagicNumber")
+class UseValueCompareFixerTest {
+  private static final String REPLACE_IS_WITH_EQUAL_SIGN =
+      "Replace `_is` operator with `=` operator";
+  private static final String REPLACE_ISNT_WITH_NOT_EQUAL_SIGN =
+      "Replace `_isnt` operator with `<>` operator";
+
+  private List<CodeAction> getCodeActions(final String code, final Range range) {
+    final MagikFile magikFile = new MagikFile(MagikFile.DEFAULT_URI, code);
+    final UseValueCompareFixer fixer = new UseValueCompareFixer();
+    return fixer.provideCodeActions(magikFile, range);
+  }
+
+  @Test
+  void testReplaceIsWithEquals1() {
+    final String code = "\"a\" _is b";
+    final Range range = new Range(new Position(0, 0), new Position(1, 0));
```

Comment:
Why the range and not the whole file/code (in other tests as well)?

Reply:
This test is inspired by https://github.com/StevenLooman/magik-tools/blob/6848a2f5f23d33df3f89db863e94da908c898400/magik-checks/src/test/java/nl/ramsolutions/sw/checks/magiktyped/fixers/TypeDocParameterFixerTest.java#L37, there we also use the Range and not the whole file.